charts/prow/README.md

```diff
@@ -0,0 +1,34 @@
+# Prow chart 
+
+This is a helm chart for the kubernetes [test-infra prow project](https://github.com/kubernetes/test-infra/tree/master/prow).  Prow is the internal CI system/workflow engine used for public kubernetes projects.  This is an attempt to the tool for use for other github projects.  Prow was written to run in it's own kubernetes cluster.
+
+## Usage
+
+###github auth tokens
+
+One goal of this chart was to minimize the manual steps around github auth tokens and webhooks.  It uses a [tool](https://github.com/jfelten) and scripts to genreate and install a github auth token as a secret in the kubernetes cluster.  The webhooks it uses are generated by a helm a pre-install hook and removed by a helm post-install hook upon deletion.
```

Comment:
Doesn't matter much to me. I'm wary of unreviewed code though, so I would prefer some sort of review. That can be accomplished by moving it to k/test-infra, or just doing a review over there.